### PR TITLE
Solve Rubocop RedundantInterpolation cops

### PR DESCRIPTION
**Pull request by zanetagebka**
#### What? Why?

- Part of [#6055](https://github.com/openfoodfoundation/openfoodnetwork/issues/6055)

#### What should we test?
Rather safe one. Just removed two cops

Cop: [RedundantInterpolation](https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/RedundantInterpolation)

#### Release notes

Changelog Category (reviewers may add a label for the release notes):

- [ ] User facing changes
- [ ] API changes (V0, V1, DFC or Webhook)
- [x] Technical changes only
- [ ] Feature toggled

The title of the pull request will be included in the release notes.

### CODE REVIEW

File: .rubocop_todo.yml [hunk 1]
```diff
@@ -1,11 +1,20 @@
...
 # versions of RuboCop, may require this file to be generated again.
 
+# Offense count: 4
+# This cop supports safe autocorrection (--autocorrect).
+# Configuration parameters: EnforcedStyle, EnforcedStyleForEmptyBraces.
+# SupportedStyles: space, no_space, compact
+# SupportedStylesForEmptyBraces: space, no_space
+Layout/SpaceInsideHashLiteralBraces:
+  Exclude:
+    - 'tmp/stimulus_reflex_installer/working/development.rb'
```
Comment (mkllnk) [inline .rubocop_todo.yml:45]: This folder should not be tracked.

File: .rubocop_todo.yml [hunk 2]
```diff
@@ -810,13 +810,14 @@ Style/ClassAndModuleChildren:
...
 
-# Offense count: 1
+# Offense count: 2
 # This cop supports unsafe autocorrection (--autocorrect-all).
 # Configuration parameters: EnforcedStyle.
 # SupportedStyles: always, always_true, never
 Style/FrozenStringLiteralComment:
   Exclude:
     - '.simplecov'
+    - 'tmp/stimulus_reflex_installer/working/development.rb'
```
Comment (mkllnk) [inline .rubocop_todo.yml:829]: This shouldn't be here.

**Review by mkllnk — CHANGES_REQUESTED** [review]
Great, the change is good but somehow a temporary file path was added. Can you remove that?

Thank you!

File: .rubocop_todo.yml (hunk 1 above)
Comment (zanetagebka) [reply .rubocop_todo.yml:45]: Sure, I will push change today later.

**Review by rioug — APPROVED** [review]
Looks good now , thanks @zanetagebka :pray: 
Note for next time, we prefer rebasing the branch on master instead of merging it, it keeps the git history cleaner.

**Review by mkllnk — APPROVED** [review]
Perfect!

**Pull request merged**